<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wacp</class>
 <widget class="QMainWindow" name="wacp">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>WACP by Lee</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>lccc.png</normaloff>lccc.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="sitelist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>911</width>
      <height>430</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>홈페이지 목록</string>
    </property>
    <widget class="QTableWidget" name="sitetable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>891</width>
       <height>385</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>20</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>이름</string>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>URL</string>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>최근 점검</string>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>점검결과</string>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>상태</string>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="home_info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>홈페이지 정보</string>
    </property>
    <widget class="QLineEdit" name="home_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="home_url">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>55</y>
       <width>401</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>32</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>이름</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_url">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>32</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_apply">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_delete">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_af">
     <property name="geometry">
      <rect>
       <x>572</x>
       <y>25</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>군내 사이트</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="check_config">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>170</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>점검 설정</string>
    </property>
    <widget class="QSpinBox" name="spin_cycle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_cycle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>점검 주기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_min">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>25</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>분</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>점검</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>중지</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
